test/integration/featureFlag.test.js

```diff
@@ -120,6 +118,8 @@ describe('FeatureFlag', function () {
         })
     })
 
+    // TODO: Add test cases to update all the possible updates to the featureFlag config
```

Comment:
Please add the ticket here for reference to track

Reply:
Ok